AWSTranscribeStreaming/AWSTranscribeStreamingService.m

```diff
@@ -309,15 +318,16 @@ - (AWSTask *)setUpWebsocketForRequest:(NSDictionary *)requestParams {
 
         NSURL *websocketURL = task.result;
         NSURLRequest *urlRequest = [NSURLRequest requestWithURL:websocketURL];
+        if (!self.webSocketProvider) { //if it wasn't initialized aka nothing was passed in then initialize it with socket rocket adaptor
+            self.webSocketProvider = [[AWSSRWebSocketAdaptor alloc] initWithURLRequest:urlRequest];
+            [self updateWebSocketDelegate];
 
-        self.webSocket = [[AWSSRWebSocket alloc] initWithURLRequest:urlRequest];
-
-        [self updateWebSocketDelegate];
+        }
 
         //Open the web socket
-        [self.webSocket open];
+        [self.webSocketProvider connect];
 
-        AWSDDLogDebug(@"webSocket %@ is created and opened", self.webSocket);
+        AWSDDLogDebug(@"webSocket %@ is created and opened", self.webSocketProvider);
```

Comment:
technically, this isn't the websocket, it's a adapter around the socket -- maybe it would be better just to mvoe this log statement to `AWSSRWebSocketAdaptor`

Reply:
moved